assets/src/components/ladder.tsx

```diff
@@ -1,28 +1,28 @@
 import React, { useContext } from "react"
 import DispatchContext from "../contexts/dispatchContext"
+import {
+  ladderVehiclesFromVehicles,
+  VehicleDirection,
+} from "../models/ladderVehicle"
 import { Timepoint, Vehicle, VehicleId, VehicleTimepointStatus } from "../skate"
 import { selectVehicle } from "../state"
 import { Orientation, Size, VehicleIconSvgNode } from "./vehicleIcon"
 
+export interface Props {
+  timepoints: Timepoint[]
+  vehicles: Vehicle[]
+  ladderDirection: LadderDirection
+  selectedVehicleId: VehicleId | undefined
+}
+
 // Timepoints come from the API in the ZeroToOne direction
 export enum LadderDirection {
   ZeroToOne,
   OneToZero,
 }
 
-export enum VehicleDirection {
-  Up,
-  Down,
-}
-
-export const vehicleDirectionOnLadder = (
-  vehicle: Vehicle,
-  ladderDirection: LadderDirection
-): VehicleDirection =>
-  (vehicle.directionId === 1) ===
-  (ladderDirection === LadderDirection.ZeroToOne)
-    ? VehicleDirection.Down
-    : VehicleDirection.Up
+export const widthOfVehicleGroup = 32
+export const heightOfVehicleGroup = 34
```

Comment:
These are only used in `ladderVehicle.ts`, and could be moved there so they can be private.